activerecord/lib/active_record/suppressor.rb

```diff
@@ -0,0 +1,55 @@
+module ActiveRecord
+  # ActiveRecord::Suppressor allows save callbacks to be suppressed within a
+  # specified block.
+  #
+  # For example, here's a pattern of creating notifications when new comments
+  # are posted. (The notification may in turn trigger an email, a push
+  # notification, or just appear in the UI somewhere):
+  # 
+  # class Comment < ActiveRecord::Base
+  #   belongs_to :commentable, polymorphic: true
+  #   after_create -> { Notification.create! comment: self,
```

Comment:
Is this a thing? Do you pass a lambda to `after_create` rather than a block now?